src/main/java/edu/cmu/cs/db/peloton/test/common/TestContainer.java

```diff
@@ -60,7 +67,16 @@ private static void testEqual(ResultSet truth, ResultSet target) throws SQLExcep
     private static List<Object> getRow(ResultSet resultSet, int columnCount) throws SQLException {
         List<Object> row = new ArrayList<>();
         for (int i = 1; i <= columnCount; i++) {
-            row.add(resultSet.getObject(i));
```

Comment:
This probably won't work as BigDecimal::valueOf uses the string representation of a double as the canonical value and that might already have round-off errors. You should use a delta comparison instead.